From 4025e855748d76080d6f8cea93fb3a8b95a76642 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 24 Feb 2018 15:36:52 +0800
Subject: [PATCH] Revert "ARM: rockchip_defconfig: remove CONFIG_PM_AUTOSLEEP"

This reverts commit e868d4ecb9fb54ea602b090bdbd10d8fbb5ea53c.

Failed to pass vintf compatibility test.

Change-Id: Iba9ee00c8bab3d917d155aeab82e7926d44387d9
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 5be4e054c38f..fff86902ef62 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -75,6 +75,7 @@ CONFIG_VFP=y
 CONFIG_NEON=y
 CONFIG_KERNEL_MODE_NEON=y
 # CONFIG_COREDUMP is not set
+CONFIG_PM_AUTOSLEEP=y
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
-- 
2.35.3

